SU-SWS/stanford-jumpstart-deployer
# updated module and profiles for jslabs and jsa

# READY FOR REVIEW

# Summary
- Updated modules and included new story page and its dependencies.

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. build jsa or jsl, prod or dev
2. verify install works.

or review travis.

# Prerequisites/Dependencies/Related
- https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/149
- https://github.com/SU-SWS/stanford_sites_jumpstart_lab/pull/6
- https://github.com/SU-SWS/stanford_sites_jumpstart_academic/pull/24
- https://github.com/SU-SWS/stanford_jumpstart/pull/77
- https://github.com/SU-SWS/stanford_carousel/pull/19
- https://github.com/SU-SWS/stanford_bean_types/pull/62
